engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -242,7 +244,7 @@ def get_users_permissions(self) -> typing.Optional[UserPermissionsDict]:
         return all_users_permissions
 
     def is_rbac_enabled_for_organization(self) -> bool:
-        _, resp_status = self.api_head(self.USER_PERMISSION_ENDPOINT)
+        _, resp_status = self.api_get(self.USER_PERMISSION_ENDPOINT)
```

Comment:
> Another small fix if to change from HEAD to GET for api/org, as there is not HEAD, according to documentation

I don't believe this change does what you intend it to do. `/api/access-control/users/permissions/search` will in fact exist as a Grafana API endpoint, under certain circumstances (ie. Grafana Enterprise and Grafana Cloud ([docs](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/access-control/))). It's used [here](https://github.com/grafana/oncall/blob/dev/engine/apps/user_management/sync.py#L50-L52)

Reply:
it does fix the error. 
I think it's [this API request](https://grafana.com/docs/grafana/latest/developers/http_api/access_control/#list-permissions-assigned-to-a-user)
Also if you check 10 lines up in file you will see another `self.api_get(self.USER_PERMISSION_ENDPOINT)` , so somebody just forgot to change that.